src/backend/wallet.rs

```diff
@@ -142,7 +142,7 @@ impl Wallet {
             None => StdRng::from_entropy(),
             Some(seed_value) => StdRng::seed_from_u64(seed_value),
         };
-        let fee = 2000;
+        let fee = 3000;
```

Comment:
There's no need as we will always be good with 1 duff per byte, 3000 refers to 3kb, the top up was over 2k bytes, because we were using MANY inputs for 15 Dash. If we really wanted we could calculated this ourselves easily. But 3000 is probably more than good enough.